tdrs-backend/tdpservice/urls.py

```diff
@@ -33,18 +33,13 @@
     path("stts/", include("tdpservice.stts.urls")),
     path("data_files/", include("tdpservice.data_files.urls")),
     path("logs/", write_logs),
-    # http://www.django-rest-framework.org/api-guide/routers/#defaultrouter
-    re_path(
-        r"^$", RedirectView.as_view(url=reverse_lazy("test_api-root"), permanent=False)
```

Comment:
This caused a 500 error on the sub path `/v1`  since there is no matching view pattern for `test_api-root`.